### ISSUE TITLE
WebBrowser title

### ISSUE THREAD

**Issue by yang4515** [expo/expo#2101]
Can I display the title of the html(Google) instead of the url(google.com) when I use Expo.WebBrowser.openBrowserAsync?
<img width="374" alt="image" src="https://user-images.githubusercontent.com/21199449/44013696-ac6b0862-9ef9-11e8-8a20-3302ca69161a.png">

**Comment by sjchmiela** [comment]
Hi, @yang4515!

`Expo.WebBrowser` uses `SFSafariViewController` native API to display the web browser on iOS. Given that information, please see the answer at StackOverflow ([_How to change SFSafariviewController's Title in swift?_](https://stackoverflow.com/a/48637095/1123156)):

> As the introduction of the documentation suggests, `SFSafariViewController` is for securely displaying web content in an app. The user can trust that he doesn't see manipulated web content. If you could change the title, you could foist pages from a wrong domain to the user and that would open the door for phishing attacks.
> 
> For this reason, Apple will not have provided any possibility to change the title.

[`SFSafariViewController` documentation](https://developer.apple.com/documentation/safariservices/sfsafariviewcontroller) also states that one of its UI features is:

> - A **read-only address field** with a security indicator and a Reader button

To sum up, for the users safety the title displayed on the bar is not changeable. 🙂
